app/models/api_docs/service.rb

```diff
@@ -101,6 +101,11 @@ def specification
     @_spec ||= ThreeScale::Swagger::Specification.new(self.body)
   end
 
+  def api_product_production_public_base_url
+    return unless service&.proxy
+    service.proxy.endpoint
+  end
```

Comment:
I love the name ❤️ 
The content can be polished later knowing that this is for 2.9 😉